### PR TITLE
Add power mode

### PR DESCRIPTION
**Pull request by pillip8282**

### CODE REVIEW

File: apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c [hunk 1]
```diff
@@ -318,7 +316,7 @@ static void itc_wifimanager_connect_ap_config_p(void)
 	printf("AP config: %s(%d), %s(%d), %d %d\n", config.ssid, config.ssid_length, config.passphrase, \
 			config.passphrase_length, config.ap_auth_type, config.ap_crypto_type);
 
-	ret =  wifi_manager_connect_ap_config(&config, &reconfig);
+	ret =  wifi_manager_connect_ap(&config)
```
Comment (jeongchanKim) [inline apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c:319]: Please add `;` at the end of the line.

File: apps/examples/wifi_manager_sample/wm_test.c [hunk 2]
```diff
@@ -606,6 +606,22 @@ void _wt_get_conn_info(void *arg)
...
+{
+	WT_ENTER;
+	struct wt_options *opt = (struct wt_options *)arg;
+	wifi_manager_powermode_e pmode = WIFI_MANAGER_POWERMODE_ENABLE;
+	if (opt->power_mode == 0) {
+		pmode = WIFI_MANAGER_POWERMODE_DISABLE;
+	}
+	wifi_manager_result_e res = wifi_manager_set_powermode(pmode);
+	if (res != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "failt to set power mode\n");
```
Comment (jeongchanKim) [inline apps/examples/wifi_manager_sample/wm_test.c:619]: Please change `failt` to `fail`.

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
It works well

**Pull request merged**